presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -956,6 +957,15 @@ public UnnestNode unnest(PlanNode source, List<VariableReferenceExpression> repl
                 ordinalityVariable);
     }
 
+    public NativeExecutionNode NativeExecution(
+            PlanNode subPlan)
+    {
+        return new NativeExecutionNode(
+                Optional.empty(),
```

Comment:
Dumb question: So NativeExecutionNode will always be the first node, right? Most of the information should be directly passed to this node?

Reply:
You're right. Unless we want to do Java and native hybrid execution in a same stage in future, the `NativeExecutionNode` will be always the top node of a fragment.